templates/microcloud/index.html

```diff
@@ -0,0 +1,444 @@
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/273d81f3-microcloud-visual-2.svg",
+              alt="",
+              width="83",
+              height="100",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+          <div class="col-3 col-medium-2 p-section--shallow">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/a8264def-microcloud-visual-3.svg",
+              alt="",
+              width="100",
+              height="100",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+          </div>
+        </div>
+        <hr class="p-rule"/>
+        <div class="row">
+          <div class="col-3 col-medium-2 p-section--shallow">
+            <h3 class="p-heading--5">Automated deployment</h3>
+            <p>MicroClouds are built with simplicity and automation in mind. With MicroClouds, both deployment and on-site operations are automated with low to no on-site maintenance, regardless of whether you're running it at the back of the office or at a hard-to-reach remote edge site.</p>
+          </div>
+          <div class="col-3 col-medium-2 p-section--shallow">
+            <h3 class="p-heading--5">Secure with hassle-free updates</h3>
+            <p>While smaller in scale, your MicroClouds will still likely process big amounts of data. They may also run sensitive workloads. Security and prompt vulnerability updates are the key priorities to ensure your data is kept safe. MicroCloud components are packaged using snaps, providing an extra layer of isolation while enabling worry-free updates.</p>
+          </div>
+          <div class="col-3 col-medium-2 p-section--shallow">
+            <h3 class="p-heading--5">Cost and resource efficient </h3>
+            <p>In addition to standard Ubuntu Server or Desktop, MicroClouds can also be run on Ubuntu Core -  a lightweight OS optimised for the edge. Combined with Ubuntu Core, users get a truly lightweight cloud, giving you resource efficiency without compromising on performance, while keeping costs at bay.</p>
+          </div>
+        </div>
+        <hr/>
+        <a class="p-button--positive" href="https://pages.ubuntu.com/rs/066-EOV-335/images/MicroCloud_Datasheet.pdf">Download the datasheet</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule"/>
+    <div class="col">
+      <h2>What is the size of a MicroCloud?</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-3">
+          <p>MicroClouds are highly available by default, so a minimum of three systems are needed.</p>
+          <p>They can easily extend in site size and scale from a small three-node to larger 50-node clusters.</p>
+        </div>
+        <div class="col-3 p-image-wrapper">
```

Comment:
```suggestion
        <div class="col-3 p-image-wrapper u-align--center">
```
This is horizontally centered in the design